src/rgw/rgw_loadgen_process.cc

```diff
@@ -60,7 +60,7 @@ void RGWLoadGenProcess::run()
 
   for (i = 0; i < num_objs; i++) {
     char buf[16 + 1];
-    gen_rand_alphanumeric(NULL, buf, sizeof(buf));
+    gen_rand_alphanumeric(cct, buf, sizeof(buf));
```

Comment:
there's a call to `append_rand_alpha(NULL, bucket, bucket, 16);` above that needs a cct also

Reply:
@cbodley Done~